plugins/inputs/net/net.go

```diff
@@ -69,6 +69,7 @@ func (n *NetIOStats) Gather(acc telegraf.Accumulator) error {
 
 		tags := map[string]string{
 			"interface": io.Name,
+			"interface_index": fmt.Sprint(interfacesByName[io.Name].Index),
```

Comment:
What is the type of `interfacesByName[io.Name].Index`? Can we please explicitly convert that to string?